api/v1alpha1/component.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2023 The Kubebb Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+const (
+	ComponentFinalizer = "kubebb.component.finalizer"
```

Comment:
IMO, use  https://github.com/kubebb/core/blob/dbb2d340af1ece61be41e0a61dae08b8b42a13ff/api/v1alpha1/common.go#L31 is enough, no need to add single finalizer for each cr.